backend/apollo/elections/serializers.py

```diff
@@ -53,46 +53,23 @@ class VoteSerializer(
 ):
     def authorize(self, attrs):
         user = self.context["request"].user
-        election = attrs["answer"].question.election
+        election = attrs["question"].election
         if not permissions.can_vote_in_election(user, election):
             raise exceptions.PermissionDenied("VOTE_UNAUTHORIZED")
 
-    answer = serializers.PrimaryKeyRelatedField(queryset=Answer.objects.all())
     author = serializers.HiddenField(default=CurrentUserDefault())
+    question = serializers.PrimaryKeyRelatedField(queryset=Question.objects.all())
 
     class Meta:
         model = Vote
-        fields = ["id", "answer", "author"]
+        fields = ["id", "answer_ciphertext", "author", "question"]
         read_only_fields = ["id"]
         validators = [
             UniqueTogetherValidator(
-                queryset=Vote.objects.all(), fields=["author", "answer"]
+                queryset=Vote.objects.all(), fields=["author", "question"]
```

Comment:
I'd remove it - we're going to support multi-choice answers.

Reply:
We can do it after vote overwriting or after adding multiple-choice questions to the frontend